### PR TITLE
Update packages9

### PR DESCRIPTION
**Pull request by kaligrafy**

### CODE REVIEW

File: packages/chaire-lib-common/src/utils/objects/__tests__/GenericPlaceCollection.test.ts [hunk 1]
```diff
@@ -13,7 +13,46 @@ import GenericPlaceCollection from '../GenericPlaceCollection';
...
+            },
+            {
+                type: 'Point', coordinates: [0, 0]
+            },
+            {
+                type: 'Point', coordinates: [0, 0]
+            }];
+        }
+    })
+}));
```
Comment (tahini) [inline packages/chaire-lib-common/src/utils/objects/__tests__/GenericPlaceCollection.test.ts:47]: let geokdbushMock = jest.fn()
around: jest.fn().mockImplementation((spatialIndex, x, y, max, res, dist) => geokdbushMock(spatialIndex, x, y, max, blabla))

Comment (kaligrafy) [reply packages/chaire-lib-common/src/utils/objects/__tests__/GenericPlaceCollection.test.ts]: fixed

**Review by tahini — CHANGES_REQUESTED** [review]

**Review by tahini — APPROVED** [review]
Beau cleanup! Et tout semble fonctionner

File: yarn.lock [hunk 2]
```diff
@@ -10447,6 +10447,14 @@ pbf@^3.0.4, pbf@^3.0.5, pbf@^3.2.1:
     ieee754 "^1.1.12"
     resolve-protobuf-schema "^2.1.0"
 
+pbf@^3.3.0:
```
Comment (tahini) [inline yarn.lock:10450]: Merge with other

**Review by tahini — CHANGES_REQUESTED** [review]

**Comment by greenscientist** [comment]
@kaligrafy , you should probably put the PR as a draft when you plan to add more commits to it.

**Review by tahini — APPROVED** [review]

**Pull request merged**